python/lsst/qserv/tests/benchmark.py

```diff
@@ -145,6 +145,9 @@ def runQueries(self, stopAt=MAX_QUERY):
                                       str(pragmas))
                     column_names = 'noheader' not in pragmas
                     sqlInterface.execute(qText, outFile, column_names)
+                    if 'sortresult' in pragmas:
+                        sortcmd = 'sort %s -o %s' % (outFile, outFile)
+                        os.system(sortcmd)
```

Comment:
Vaikunth is looking into switching to sorting in python, no worries :). And sure, checking the return status from sort is useful.